apps/dashboard/src/components/primitives/tag-input.tsx

```diff
@@ -107,9 +113,7 @@ const TagInput = forwardRef<HTMLInputElement, TagInputProps>((props, ref) => {
               <CommandGroup>
                 {inputValue !== '' && (
                   <CommandItem
-                    // We can't have duplicate keys in our list so adding a prefix
-                    // here to differentiate this from a possible suggestion value
-                    value={`input-${inputValue}`}
+                    value={`${inputValue}`}
```

Comment:
Can we simplify to just the value now?
```suggestion
                    value={inputValue}
```

Reply:
👁️